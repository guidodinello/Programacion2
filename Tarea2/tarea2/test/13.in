# Implementar cambiarEnCadena, intercambiar
insertarSegmentoDespues 3 (5,3.4) (8,9.4) (9,2.6)
inicioCadena
cambiarEnCadena (3,2.5)
imprimirCadena
siguiente
cambiarEnCadena (6,9.2)
imprimirCadena
siguiente
cambiarEnCadena (7,2.5)
imprimirCadena
intercambiar 1 1
imprimirCadena
intercambiar 3 3
imprimirCadena
finalCadena
infoCadena
intercambiar 3 1
infoCadena
imprimirCadena
intercambiar 1 3
infoCadena
imprimirCadena
intercambiar 2 3
infoCadena
imprimirCadena
Fin
