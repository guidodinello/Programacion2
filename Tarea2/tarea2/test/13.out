1>#  Implementar cambiarEnCadena, intercambiar.
2>Segmento insertado despuÃ©s de loc.
3>loc al inicio.
4>Cambio.
5>(3,2.50)(8,9.40)(9,2.60)
6>loc al siguiente.
7>Cambio.
8>(3,2.50)(6,9.20)(9,2.60)
9>loc al siguiente.
10>Cambio.
11>(3,2.50)(6,9.20)(7,2.50)
12>Intercambio.
13>(3,2.50)(6,9.20)(7,2.50)
14>Intercambio.
15>(3,2.50)(6,9.20)(7,2.50)
16>loc al final.
17>(7,2.50)
18>Intercambio.
19>(3,2.50)
20>(7,2.50)(6,9.20)(3,2.50)
21>Intercambio.
22>(7,2.50)
23>(3,2.50)(6,9.20)(7,2.50)
24>Intercambio.
25>(6,9.20)
26>(3,2.50)(7,2.50)(6,9.20)
27>Fin.
